modules/util/css-builder/build.gradle

```diff
@@ -59,9 +75,45 @@ liferay {
 	deployDir = "../../../tools/sdk/dependencies/com.liferay.css.builder/lib"
 }
 
+publish {
+	dependsOn buildPluginDescriptor
+	dependsOn jar
+}
+
+publishing {
+	publications {
+		maven(MavenPublication) {
+
+			groupId 'com.liferay'
+			artifactId jar.baseName
+			version "${version}"
+			from components.java
+		}
+	}
+	repositories {
+		maven {
+
+			url uri(repoPath)
+		}
+	}
+
+}
+
 test {
 	dependsOn copyPortalCommonCSSJar
+	dependsOn publish
 	dependsOn unzipPortalCommonCSS
+
+	doFirst {
+		systemProperty "maven.distribution.file.name", configurations.mavenDistribution.singleFile
+	}
+
+	systemProperty 'cssbuilder.repolocation', "${repoPath}"
+	systemProperty 'cssbuilder.version', "${version}"
+
+	doLast {
```

Comment:
This wont be needed once we generate the repo into a build/test type output folder.